src/Behat/Behat/Gherkin/Specification/Locator/FilesystemFeatureLocator.php

```diff
@@ -77,15 +77,15 @@ public function locateSpecifications(Suite $suite, $locator)
         if ($locator) {
             $filters = array(new PathsFilter($suiteLocators));
 
-            return new LazyFeatureIterator($suite, $this->gherkin, $this->findFeatureFiles($locator), $filters);
+            return new LazyFeatureIterator($suite, $this->gherkin, $this->findFeatureFiles($locator), $filters, $this->basePath);
```

Comment:
Note, you're passing in only **4** arguments here. The constructor expects **5**, you'll want to include the `$filters` parameter here.